pkg/datapath/linux/route/route_linux.go

```diff
@@ -190,7 +239,21 @@ func replaceRoute(route Route, mtuConfig mtu.Configuration) (bool, error) {
 	}
 
 	if lookup(link, &routeSpec) == nil {
-		if err := netlink.RouteReplace(&routeSpec); err != nil {
+		err := fmt.Errorf("routeReplace not called yet")
```

Comment:
On which occasion would this exact message occur?

Reply:
Never. I figured that it is better to have a unique message appear that we can grep for *if* it ever happens than nil which makes it impossible to debug.